src/settings/JobProfiles/JobProfileDetailsActionMenu/JobProfileDetailsActionMenu.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Button,
+  Icon,
+} from '@folio/stripes/components';
+
+const JobProfileDetailsActionMenu = ({
```

Comment:
As far I understand, you are going to add the action menu for the mapping profile view page in the scope of UIDEXP-75. They look very similar according to the stories' descriptions. Should the shared component be created for these purposes?

Reply:
Yep, this what I am actually planning to do. To move eventually this component to the shared components. If you think it makes sense doing now, I am okay with that too.

Reply:
Only clarified. I think, you will do all the best. So you can do it in the next ticket.